src/APIFunctions/3DPrinting.js

```diff
@@ -123,16 +137,18 @@ export async function update3DPrintRequestProgress (requestToUpdate, token) {
  * @returns {(Object[]|string)} - If the request was sucessfully, return array of requests.
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * request or the search result
```